maps/southern_cross/southern_cross-3.dmm

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
Bot can you _fucking not_?

**HOW IS THERE A CONFLICT?! WE DON'T EVEN TOUCH THIS FILE!!!**